### PR TITLE
Use beginning of day for week_creation_chart count

### PR DESCRIPTION
**Pull request by sethherr**

### CODE REVIEW

File: app/models/counts.rb [hunk 1]
```diff
@@ -45,7 +49,8 @@ def assign_recoveries
     end
 
     def assign_week_creation_chart
-      assign_for("week_creation_chart", Bike.unscoped.where(created_at: (Time.now - 7.days)..Time.now).group_by_day(:created_at).count.to_json)
+      Time.zone = TimeParser::DEFAULT_TIMEZONE # Just to make sure
+      assign_for("week_creation_chart", Bike.unscoped.where(created_at: beginning_of_week..Time.now).group_by_day(:created_at).count.to_json)
```
Comment (jmromer) [inline app/models/counts.rb]: Defaulting to `Time.current` obviates the need for this

Comment (sethherr) [reply app/models/counts.rb]: hahahaha. You got me!

Comment (sethherr) [reply app/models/counts.rb]: Updated in c5f09030b44970e762149d07e74585a5b6ea7bde

Comment (jmromer) [reply app/models/counts.rb]: There's still a `Time.now` on line 52. I'm 99.9% sure it doesn't make a difference given (1) the TZ we set on the server, (2) the TZ set in the Rails app, and (3) the figures being computed here, but in general it takes some cognitive load off to not have to reason through it every time :)

Comment (sethherr) [reply app/models/counts.rb]: Yeah yeah yeah, #892

**Pull request merged**